### PR TITLE
Chat

### PR DESCRIPTION
**Pull request by shreyala**

### CODE REVIEW

File: templates/home.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <title>SayLess - Home</title>
   <!-- Bootstrap CSS and fa-icon -->
   <link rel="stylesheet" href="/include/bootstrap.css">
-  <link rel="stylesheet" href="/home.css">
+  <link rel="stylesheet" href="LOCAL homepage.css">
```
Comment (Shazmaan) [inline templates/home.html]: Can you please change this as this will break the site.

File: templates/home.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <title>SayLess - Home</title>
   <!-- Bootstrap CSS and fa-icon -->
   <link rel="stylesheet" href="/include/bootstrap.css">
-  <link rel="stylesheet" href="LOCAL homepage.css">
+  <link rel="stylesheet" href="/statics/home.css">
```
Comment (Shazmaan) [inline templates/home.html]: No need to have statics. Keep it as /home.css

File: templates/home.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
   <script src = "https://code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
   <script src="/include/bootstrap.js"></script>
   <script src="/include/popper.min.js"></script>
-  <script src="/home.js"></script>
+  <script src="/statics/home.js"></script>
```
Comment (Shazmaan) [inline templates/home.html]: No need to have statics. Keep this as /home.js

File: templates/home.html (hunk 2 above)
Comment (shreyala) [reply templates/home.html]: Done

File: templates/home.html (hunk 3 above)
Comment (shreyala) [reply templates/home.html]: Done

**Review by Shazmaan — APPROVED** [review]

**Pull request merged**